src/NoOpContainer.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+
+namespace Dhii\Container;
+
+
+use ArrayIterator;
+use Dhii\Collection\ClearableContainerInterface;
+use Dhii\Collection\MutableContainerInterface;
+use Dhii\Collection\WritableContainerInterface;
+use Dhii\Collection\WritableMapInterface;
+use Dhii\Container\Exception\ContainerException;
+use Dhii\Container\Exception\NotFoundException;
+use IteratorAggregate;
+
+class NoOpContainer implements
```

Comment:
Maybe add phpdoc.

Like in my quick fix PR

```
/**
 * A container that does nothing.
 * This can be used if an actual implementation is not available,
 * without extra checks or nullables - just as if it was a real one.
 */
```